### ISSUE TITLE
Tags

### ISSUE THREAD

**Issue by b-allard** [clarklab/chowdown#35]
Hello,

I have created several recipes with tags.
I thought it was possible to search with tags but apparently it is not the case.

Can you tell me what the tags are used for and how I can use them to search with?

Thank you in advance, 

A new chowndown user already convinced :)

PS: Thanks for all the work done on the framework :) It looks great!

**Comment by MichaelThomasMPT** [comment]
Hey @b-allard - I added the ability to search by tags in PR #28 from early February. Is the version you're running more recent than that? 

In case it's to do with the tags in your recipes themselves, [here's an example](https://raw.githubusercontent.com/MichaelThomasMPT/chowdown/my-recipes/_recipes/apple-rhubarb-crumble.md), for comparison with your own, of a recipe where I've added two tags, both of which work with the search. 

Hope this helps!
